relay-dynamic-config/src/feature.rs

```diff
@@ -80,14 +80,12 @@ pub enum Feature {
     /// Serialized as `organizations:continuous-profiling-beta-ingest`.
     #[serde(rename = "organizations:continuous-profiling-beta-ingest")]
     ContinuousProfilingBetaIngest,
-    /// Enables metric extraction from spans for common modules.
-    ///
-    /// Serialized as `projects:span-metrics-extraction`.
+    /// This feature has graduated and is hard-coded for external Relays.
+    #[doc(hidden)]
     #[serde(rename = "projects:span-metrics-extraction")]
     ExtractCommonSpanMetricsFromEvent,
-    /// Enables metric extraction from spans for addon modules.
-    ///
-    /// Serialized as `projects:span-metrics-extraction-addons`.
+    /// This feature has graduated and is hard-coded for external Relays.
```

Comment:
Graduated features are hardcoded at the top of the file in the `GRADUATED_FEATURE_FLAGS` slice, but I don't think we can graduate it, as that would unconditionally enable it.

Should call out that this is deprecated and just kept around for external Relays, we can also maybe just graduate them if the span extraction one is enabled, then we can remove them from Sentry.